flake.nix

```
@@ -35,6 +35,10 @@
         devShells.default = mkShell {
           packages = [
             alejandra
+            bashInteractive
+            coreutils
+            hyperfine
+            just
```

```suggestion
            just
            cargo-bloat
            cargo-audit
            cargo-udeps
```
Probably should add these extra cargo packages used in various places of the justfile.